source/_integrations/miele.markdown

```diff
@@ -0,0 +1,155 @@
+
+{% enddetails %}
+
+{% include integrations/config_flow.md %}
+
+The integration configuration will ask for the *Client ID* and *Client Secret* created above. See [Application Credentials](/integrations/application_credentials) for more details.
+
+## Supported functionality
+
+{% note %}
+
+- The entities' availability depends on the appliance type and the generation of the product, and the appliance might not support all the entities for its type.
+- Products from professional and semi-professional series are generally not supported due to the limitations in the Miele 3rd party API.
+- Some appliances don't report data while they are turned off, so corresponding entities will not appear in the Miele integration after loading until the appliances are turned on.
+{% endnote %}
+
+### Sensor
+
+{% details "List of sensors" %}
+
+- **Operation state**:
+  - **Status**: Represents the current operation state of the device. The default entity name is just the appliance type e.g., "Dishwasher".
+  - **Temperature**: Represents the current temperature in refrigerators, freezers and ovens. Entities are created for up to 3 zones depending on the device capabilities.
+{% enddetails %}
+
+## Automation examples
+
+Get started with these automation examples
+
+### Send a notification when the appliance ends the program
+
+{% details "Example YAML configuration" %}
+
+{% raw %}
+
+```yaml
+alias: "Notify when program ends"
+triggers:
+  - trigger: state
+    entity_id:
+      - sensor.washing_machine
+    to: program_ended
+actions:
+  - service: notify.notify
+    data:
+      message: "The appliance has finished the program."
+```
+
+{% endraw %}
+{% enddetails %}
+
+## Data updates
+
+This integration uses server-sent events from the Miele API to receive live updates from the appliances.
+When the configuration entry is loaded or after a streaming error (for example after disconnection), the integration will request all data (such as appliance info, available commands, programs, settings, and status) for all appliances.
+If a new appliance is added to the account, the integration will request data for the new appliance and expose the related entities automatically after a reload of the integration.
+
+## Known limitations
+
+- The Miele 3rd party API does not fully match the Miele app. Some programs, options, or settings available in the app may not be accessible or usable via the API.
+- This integration supports only one integration entry, as the Miele 3rd party API does not allow for the unique identification of an account.
+
+## Troubleshooting
+
+### Unavailable entities for a device
+
+#### Symptom: "The entities related to an appliance were available but no longer are"
+
+After reloading the Miele integration, the entities related to an appliance that used to be available are no longer available.
+Also, when downloading the diagnostics data from the device page, the technical data is obtained:
```

Comment:
Why does this last sentence sound like something is coming but then nothing comes related to the first part of the sentence